internal/api/grpc/management/user_grant_converter.go

```diff
@@ -139,25 +139,26 @@ func userGrantViewFromModel(grant *grant_model.UserGrantView) *management.UserGr
 	logging.Log("GRPC-lpsg5").OnError(err).Debug("unable to parse timestamp")
 
 	return &management.UserGrantView{
-		Id:            grant.ID,
-		State:         usergrantStateFromModel(grant.State),
-		CreationDate:  creationDate,
-		ChangeDate:    changeDate,
-		Sequence:      grant.Sequence,
-		ResourceOwner: grant.ResourceOwner,
-		UserName:      grant.UserName,
-		FirstName:     grant.FirstName,
-		LastName:      grant.LastName,
-		Email:         grant.Email,
-		ProjectName:   grant.ProjectName,
-		OrgName:       grant.OrgName,
-		OrgDomain:     grant.OrgDomain,
-		RoleKeys:      grant.RoleKeys,
-		UserId:        grant.UserID,
-		ProjectId:     grant.ProjectID,
-		OrgId:         grant.ResourceOwner,
-		DisplayName:   grant.DisplayName,
-		GrantId:       grant.GrantID,
+		Id:              grant.ID,
+		State:           usergrantStateFromModel(grant.State),
+		CreationDate:    creationDate,
+		ChangeDate:      changeDate,
+		Sequence:        grant.Sequence,
+		ResourceOwner:   grant.ResourceOwner,
+		UserName:        grant.UserName,
+		FirstName:       grant.FirstName,
+		LastName:        grant.LastName,
+		Email:           grant.Email,
+		ProjectName:     grant.ProjectName,
+		OrgName:         grant.OrgName,
+		OrgDomain:       grant.OrgDomain,
+		RoleKeys:        grant.RoleKeys,
+		UserId:          grant.UserID,
+		ProjectId:       grant.ProjectID,
+		OrgId:           grant.ResourceOwner,
+		DisplayName:     grant.DisplayName,
+		GrantId:         grant.GrantID,
+		UserDescription: grant.Description,
```

Comment:
why user description not description?

Reply:
I wanted to clarify that its the description of the user and not the org. I can also rename to description

Reply:
Ok. Yes if we need it this is better. but do we really need description on the grant? in my opinion we just show loginname and display name